app/src/main/java/fr/free/nrw/commons/category/CategoryDetailsViewModel.kt

```diff
@@ -0,0 +1,76 @@
+package fr.free.nrw.commons.category
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.viewModelScope
+import fr.free.nrw.commons.bookmarks.category.BookmarkCategoriesDao
+import fr.free.nrw.commons.bookmarks.category.BookmarksCategoryModal
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.update
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+class CategoryDetailsViewModel(
```

Comment:
Can you add kdoc to all new classes? And ideally functions too. Thanks! :-)

Reply:
My bad :(  I was going to but forgot. Doing it now.